test/statsd/ecs_resources/ec2_launch/daemon/ecs_taskdef.tpl

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "name": "cloudwatch_agent",
+    "image": "${cwagent_image}",
+    "essential": true,
+    "secrets": [
+      {
+        "name": "CW_CONFIG_CONTENT",
+        "valueFrom": "${cwagent_ssm_parameter_arn}"
+      },
+      {
+        "name": "PROMETHEUS_CONFIG_CONTENT",
```

Comment:
Do we still need the prometheus config for this? Is this being reused by our prometheus testing?